### PR TITLE
Create api_extension(update)2

### LINKED ISSUE #724: Add mentorship: flutter and ios client to API Swagger description
### Description
As a contributor,
I need information accessible in the Swagger UI,
so that I can get to know more about mentorship system projects.

The idea is to add an extra line mentioning the mentorship : flutter client and ios-client and its URL for both

### Mocks
This is how the description looks like at the moment:

image

### Acceptance Criteria

- [ ] Update [Required]
- [ ]  Add ios client text on Swagger UI description
- [ ] Add flutter client text on Swagger UI description

### Definition of Done

- [ ]  All of the required items are completed.
- [ ]  Approval by 1 mentor.

### Estimation
30 min

### PR DESCRIPTION
**Pull request by 19-pragya**
### Description

Fixes #724

### Type of Change:

- Code

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged
- [ ] Update Postman API at /docs folder
- [ ] Update Swagger documentation and the exported file at /docs folder
- [ ] Update requirements.txt

**Code/Quality Assurance Only**

- [ ] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Review by vj-codes — CHANGES_REQUESTED** [review]
@19-pragya why is everything from title repeated twice?Just remove it once, leave a line after `from flask_restx import Api` and shift the lines you added below the Android client line.
Just these minor changes and then , you're done!
Great work on your first PR here btw:)

**Comment by 19-pragya** [comment]
@vj-codes  please check now!

**Review by vj-codes — CHANGES_REQUESTED** [review]
Great process , sorry but title,version and description, docs is still repeated twice.And shift line flutter and ios line just below line 9 .
Please change this last time and then you're done @19-pragya  👍

**Comment by 19-pragya** [comment]
@vj-codes please check
https://github.com/anitab-org/mentorship-backend/pull/725/files

**Review by anna4j — APPROVED** [review]

**Comment by anna4j** [comment]
@19-pragya LGTM! Check out Open Source Hack:  https://anitab-org.github.io/events/open-source-hack/

@vj-codes Should I close this issue? Please review it

**Comment by vj-codes** [comment]
@anna4j I was waiting for the author to shift the Flutter and ios lines to be shifted just below the Android one to maintain consistency. I mentioned it twice earlier so just waiting:)

**Comment by isabelcosta** [comment]
@19-pragya I updated your PR description to include the link. Please take a look :)
Also, you need to fill in a little more information, even if this is a very small PR, its good if you show us how you tested, to help reviewers understand how they can also test your change, and check the checklist for what items you completed :)

I will review your PR now 🤗

File: api_extension(update) [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    title="Mentorship System API",	    
+    version="1.0",	    
+    description="API documentation for the backend of Mentorship System. \n \n"	   
+    + "Mentorship System is an application that matches women in tech to mentor each other, on career development, "	   
+    + "through 1:1 relations during a certain period of time. \n \n"	   
+    + "The main repository of the Backend System can be found here: https://github.com/anitab-org/mentorship-backend \n \n"	   
+    + "The Android client for the Mentorship System can be found here: https://github.com/anitab-org/mentorship-android \n \n"
+    + "For more information about the project here's a link to our wiki guide: https://github.com/anitab-org/mentorship-backend/wiki"	   
+    + "For flutter client for the Mentorship System can be found here: https://github.com/anitab-org/mentorship-flutter"
+    + "For ios client for the Mentorship System can be found here: https://github.com/anitab-org/mentorship-ios"
```
Comment (isabelcosta) [inline api_extension(update):12]: These 2 lines are correct :) but the file you put them on is not. You are creating a new file, when what its desired is to edit a specific file called `/app/api/api_extension.py` and this is where you have to add those 2 lines.

Line: https://github.com/anitab-org/mentorship-backend/blob/develop/app/api/api_extension.py#L11

To test this change you have to run the project, so please follow the instructions on the README to do so, and then provide a screenshot of your successful change showing it :)

**Review by isabelcosta — CHANGES_REQUESTED** [review]
@19-pragya thank you for working on this issue 🎉  this will really enhance the Swagger UI :)

I requested a change to make your contribution match what the issue is requesting, after that is done I am more than happy to approve!

**Comment by vj-codes** [comment]
Closing as flutter and ios have been added currently

**Pull request closed without merging**